components/input_environmental_personal.py

```diff
@@ -291,6 +332,13 @@ def input_environmental_personal(
     )
 
     inputs.append(unit_toggle)
+    # show custom ensemble button
+    custom_ensemble_button = None
+    if selected_model in ["PMV_EN", "PMV_ashrae"]:
```

Comment:
again, I said this many many many times, never use strings but use the value from the class

Reply:
Sorry this should not be committed, I forgot to delete this part of the code

Reply:
I fixed the code like this `if selected_model in [Models.PMV_EN.name, Models.PMV_ashrae.name]:`

Reply:
Its been fixed like this way 
![image](https://github.com/user-attachments/assets/5af65683-2e23-46af-a38d-d18e59614e2f)